flixel/text/FlxText.hx

```diff
@@ -942,7 +962,7 @@ class FlxText extends FlxSprite
 	{
 		var graphic:BitmapData = _hasBorderAlpha ? _borderPixels : graphic.bitmap;
 		_matrix.translate(x, y);
-		graphic.draw(textField, _matrix);
+		drawTextFieldTo(graphic);
```

Comment:
We won't remove the border styles any time soon, that would be a big breaking change.